Cogs.Tests/Cogs.Tests.csproj

```diff
@@ -7,6 +7,18 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <None Remove="cogsburger.zip" />
+    <None Remove="dotnet.exe" />
+    <None Remove="normativeXMI.xsd" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Include="cogsburger.zip" />

```

Comment:
git ignore the zip file that is created for the tests.
Add a pre-build event for the test project to zip the cogsburger directory